discord-bot/README.md

```diff
@@ -0,0 +1,42 @@
+# discord-bot
+
+This sample demonstrates using Mongoose to build a sample 
+[Discord](https://discord.com) bot.
+
+## Setup
+
+Make sure you have a local Stargate instance running as described on the [main page](../README.md) of this repo.
+
+## Running This Example
+
+1. Create a `.env` file with the keys from `.env.example` and the values from the developer portal and your Discord server.*
+1. Run npm install
+1. Run node ./deploy-commands.js
+1. Run node ./index.js
+
+* `DISCORD_GUILD_ID`: go into your server's settings and click on "Widget". The "Server ID" is the `guildId`.
+* `DISCORD_CLIENT_ID`: go to the [Discord developer portal](https://discord.com/developers/applications), click your bot, and then click "OAuth2". The "Application ID" is the `clientId`.
+* `DISCORD_TOKEN`: go to the [Discord developer portal](https://discord.com/developers/applications), click your bot, and then click on "Bot".
+* Make sure to enable slash commands when selecting bot permissions.
```

Comment:
I would suggest elaborate this a little bit more something like below. I had to spend some time figuring out this, so I thought this might be helpful for others as well, which will improve the getting started experience.

> "Navigate to OAuth2 -> URL Generator", select 'bot' in the SCOPES and 'Use Slash Commands' in the 'BOT PERMISSIONS' section, copy the URL that is generated, go to the URL on a new table and authorize as required"

Reply:
Thanks! I didn't do as much of a write-up as I should have the first time I walked through the Discord bot setup.

Reply:
Got it! BTW, the screenshots you added were very helpful for me (as the first time creator of a discord bot).